### PR TITLE
Update jurisdiction metadata downloads list

### LINKED ISSUE #1498: Make it possible to upload and download jurisdiction structure counts
- [ ] Upload and download structure count jurisdiction metadata (`jurisdiction_metadata-structures`)

### PR DESCRIPTION
**Pull request by ciremusyoka**
Closes #1498 
Add structures to the list of jurisdiction metadata items that can downloaded

### CODE REVIEW

File: src/constants.tsx [hunk 1]
```diff
@@ -254,6 +254,7 @@ export const JURISDICTION_METADATA_RISK = 'jurisdiction_metadata-risk';
 export const JURISDICTION_METADATA_COVERAGE = 'jurisdiction_metadata-coverage';
 export const JURISDICTION_METADATA_POPULATION = 'jurisdiction_metadata-population';
 export const JURISDICTION_METADATA_TARGET = 'jurisdiction_metadata-target';
+export const JURISDICTION_METADATA_STUCTURES = 'jurisdiction_metadata-structures';
```
Comment (moshthepitt) [inline src/constants.tsx:257]: Does the data warehouse query expect this to be `jurisdiction_metadata-structures` or `jurisdiction_metadata-structure`?  I think its helpful to provide a link to one of the queries that use this on the Reveal datawarehouse repo.

Comment (ciremusyoka) [reply src/constants.tsx:257]: It expects `jurisdiction_metadata-structures`

Comment (moshthepitt) [reply src/constants.tsx:257]: My point is lets link that here to serve as documentation.

Comment (ciremusyoka) [reply src/constants.tsx:257]: Here is a link https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/32ac211b78a3d4cd3ada4f3f7e55024a06dd57b3/3-reveal/migrations/8-IRS-Lite/1-Zambia-2020/deploy/remove_hard_coded_jurisdiction_depth.psql#L154

**Review by moshthepitt — APPROVED** [review]

**Review by kelvin-muchiri — APPROVED** [review]

**Pull request merged**